ci/pipelines/deploy-to-test.yml

```diff
@@ -635,7 +635,26 @@ jobs:
       - get: pay-ci
       - load_var: carbon_relay_image_tag
         file: carbon-relay-ecr-registry-test/tag
-      #TODO - task: create notification snippets  
+      - task: create-failure-notification-snippet
+        config:
+          platform: linux
+          image_resource:
+            type: docker-image
+            source:
+              repository: alpine
+          params:
+            CARBON_RELAY_IMAGE_TAG: ((.:carbon_relay_image_tag))
+          outputs:
+            - name: snippet  
+          run:
+            path: sh
+            args:
+              - -c
+              - |
+                echo "- <https://github.com/alphagov/pay-dockerfiles/releases/tag/${CARBON_RELAY_IMAGE_TAG} v${CARBON_RELAY_IMAGE_TAG}>" | \
+                tee -a snippet/failure || true
```

Comment:
On the apps' `create-notification-snippets.yml` task we include a standard first line as well (`[emoji] FARGATE blah blah blah`). I think this would miss that out? Can we test this (maybe in `#govuk-pay-activity` to reduce noise)?

Reply:
Ok the notification would look like [this](https://gds.slack.com/archives/CNZFSNM0S/p1624875712015800). It was generated by appending the code to the end of the `deploy-carbon-relay` job (but obviously it will not be in the PR).
```    on_success:
      put: slack-notification
      params:
        channel: '#govuk-pay-starling'
        text: "((.:failure_snippet)) \n
              - <https://cd.gds-reliability.engineering/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|Concourse build #$BUILD_NAME>"
```

Reply:
Ah lovely, as long as it's consistent with the others I'm happy.